docs/source/tuning_strategies.md

```diff
@@ -145,37 +145,74 @@ conf = PostTrainingQuantConfig(
 
 ### Design
 
-The `Basic` strategy is designed for quantizing most models. There are six stages executed by `Basic` strategy sequentially, and the tuning process ends once the condition meets the exit policy. 
+The `Basic` strategy is designed for quantizing most models. There are several stages executed by `Basic` strategy sequentially, and the tuning process ends once the condition meets the exit policy.  The diagram below illustrates each stage, accompanied by additional details provided for each annotated step.
+
+
+
+```mermaid
+flowchart TD
+	classDef circleText text-anchor:middle;
+	start([Start])
+	s1("1. Default quantization")
+	s2("2. Apply all recipes [Opt]")
+	s3("3. OP-type-wise tuning")
+	s4("4. Try recipe one by one [Opt]")
+	s5("5.1 Block-wise fallback*")
+	s6("5.2 Instance-wise fallback")
+	s7("5.3 Accumulated fallback")
+	
+	start --> s1
+	s1 --> s2:::circleText
+	s2 --> s3
+	s3 --> s4
+	s4 --> s5
+	subgraph "     Fallback"
+	s5 --> s6
+	s6 --> s7
+	end
```

Comment:
Since the tuning process exits once it meets the exit policy, I don't have a good idea of how to depict it in the diagram. However, I did mention it in the overall [introduction](https://github.com/intel/neural-compressor/blob/ly/block_doc/docs/source/tuning_strategies.md#design-2).